zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/AuthorizationCheckBehavior.java

```diff
@@ -56,13 +62,43 @@ public final class AuthorizationCheckBehavior {
   private final boolean authorizationsEnabled;
   private final boolean multiTenancyEnabled;
 
+  private final LoadingCache<Map<String, Object>, AuthorizedTenants> tenantIdCache;
+  private final LoadingCache<AuthorizationRequest, Set<AuthorizationScope>> authorizationScopeCache;
```

Comment:
🔧 It feels weird to use the `AuthorizationRequest` as the key for a cache as it contains a command. I understand for job push it won't as there is no command, but the cache is not limited to just job push.